src/main/java/com/svyrydova/modul2/models/Invoice.java

```diff
@@ -0,0 +1,12 @@
+package com.svyrydova.modul2.models;
+
+import java.util.ArrayList;
+
+import com.svyrydova.modul2.service.ShopService;
+import lombok.Data;
+
+@Data
+public class Invoice {
+    private final ArrayList<Sale> sales = ShopService.readCsv("saleList.csv");
```

Comment:
Reading from a file should be more explicit, called through a method, for example, in a main